docs-src/tut-7-rpc.scrbl

```diff
@@ -24,12 +24,39 @@ Reach code.
 Below we will compare the @secref["tut-7"] JavaScript @tech{frontend} with the
 equivalent Python code communicating via RPC, section by section.
 
-First, we need to install the library from the command line:
-@cmd{
-  pip install --upgrade reach-rpc-client
+@(hrule)
+
+First we need to set up our project and install the library from the command
+line:
+
+@verbatim{
+$ mkdir rps-python
+$ cd rps-python
```

Comment:
I think you should drop these two things because we haven't been doing that the whole time